client/packages/cli/src/schemas/xcm.ts

```diff
@@ -10,27 +10,32 @@ export const XcmTransferSchema = z.object({
       invalid_type_error: 'XCM transfer type must be a string',
       required_error: 'XCM transfer type is required',
     })
-    .max(6),
+    .regex(/^(relay|system|para)$/),
   endpoint: z.string({
     invalid_type_error: 'Enpoint must be a string',
     required_error: 'Endpoint is required',
-  }),
+  }).startsWith('ws://'),
```

Comment:
ws or wss

Reply:
Updated  the value to be an enum in commit [40a646a](https://github.com/t3rn/t3rn/pull/1444/commits/40a646a0d3d0d939c061237cf09ec900b8dd6553)